src/config/config.ts

```diff
@@ -1,14 +1,13 @@
-import Web3 from "web3";
+import { ethers } from "ethers";
 
 import MemoryQueue from "../batch/memoryQueue";
 import { QueueInterface } from "../batch/queue";
 import MemoryStore from "../storage/memoryStorage";
 import { StorageInterface } from "../storage/storage";
-import { EthereumAddress } from "../types/Strings";
 
 export interface Config {
-  accountAddress?: EthereumAddress;
-  provider?: Web3;
+  provider?: ethers.providers.Provider;
+  signer?: ethers.Signer;
```

Comment:
It doesn't necessarily need to happen in this PR, but now that we're using ethers, I wonder if we can remove the optionality from these keys and include a default ethers provider in the init below.